tests/scf-hess4/input.dat

```diff
@@ -0,0 +1,39 @@
+#! CONV SCF 6-31G analytical vs finite-difference tests
+#! Tests UHF hessian code for Ca != Cb
```

Comment:
Copy-paste error? The orbitals are the same

Reply:
Yes, good catch. Fixed here and in scf-hess5